pkgs/servers/trinitycore/update.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p nix-prefetch-git -p jq -p python3 -i python3
+from subprocess import run, PIPE
+import fileinput
+import json
+import re
+
+
+owner = "vmangos"
+repo = "core"
```

Comment:
This is obviously the wrong incomplete update script, but we really should have one, given that we require the commit rev to embed into the binaries.

Reply:
The update script is slightly better now.